### PR TITLE
Introduce the llvm13_broken_tests option and disable failing tests with LLVM 13 on Windows

### PR DESCRIPTION
**Pull request by vgvassilev**

### CODE REVIEW

File: python/cling/CMakeLists.txt [hunk 1]
```diff
@@ -14,13 +14,15 @@ if(ROOT_pyroot_FOUND)
...
   # https://bitbucket.org/wlav/cppyy/issues/65
   # For now, we rely on our own implementation of TPython
-  if(NOT MSVC OR "${CMAKE_GENERATOR_PLATFORM}" MATCHES "Win32" OR win_broken_tests)
-    ROOTTEST_ADD_TEST(class MACRO
-                      runPyClassTest.C
-                      WORKING_DIR ${CMAKE_CURRENT_SOURCE_DIR}
-                      OUTREF PyClassTest.ref
-                      ENVIRONMENT CLING_STANDARD_PCH=none
-                      CPPYY_BACKEND_LIBRARY=${CMAKE_BINARY_DIR}/lib/libcppyy_backend${PYTHON_UNDER_VERSION_STRING_Development_Main}.so)
+  if(NOT MSVC OR ${LLVM_VERSION} VERSION_LESS 13.0.0 OR llvm13_broken_tests)
```
Comment (pcanal) [inline python/cling/CMakeLists.txt:17]: Is `LLVM_VERSION` defined in standalone build of roottest?

**Comment by pcanal** [comment]
How does this (similar looking PR) relate to https://github.com/root-project/roottest/pull/915 ?

**Comment by pcanal** [comment]
@phsft-bot build
to see if the error are transient ...

File: python/cling/CMakeLists.txt (hunk 1 above)
Comment (bellenot) [reply python/cling/CMakeLists.txt:17]: I hope so, I'll cross-check later

**Comment by bellenot** [comment]
> How does this (similar looking PR) relate to #915 ?

It's the same (we had a chat with Vassil), I'll merge the #915

**Comment by pcanal** [comment]
Well something is odd ... Ah I see .. because of the branch name, this one is failing because it is using `llvm13` while the older one is not ... hence the difference in test failure.

File: python/cling/CMakeLists.txt (hunk 1 above)
Comment (bellenot) [reply python/cling/CMakeLists.txt:17]: Good point, `LLVM_VERSION` is not defined. @vgvassilev is there a way of knowing the LLVM version used by ROOT?

**Comment by vgvassilev** [comment]
> > How does this (similar looking PR) relate to #915 ?
> 
> It's the same (we had a chat with Vassil), I'll merge the #915

Ok, if that's the preference, let's close this one.

**Pull request closed without merging**